sphinx/util/texescape.py

```diff
@@ -49,6 +43,22 @@
     # OHM SIGN U+2126 is handled by LaTeX textcomp package
 ]
 
+# A map to avoid TeX ligatures or character replacements in PDF output
+# xelatex/lualatex/uplatex are handled differently (#5790, #6888)
+ascii_tex_replacements = [
+    # Note: the " renders curly in OT1 encoding but straight in T1, T2A, LY1...
+    #       escaping it to \textquotedbl would break documents using OT1
+    #       Sphinx does \shorthandoff{"} to avoid problems with some languages
+    # Can't replace - by {}- because it breaks URLs
+    # ('-', '{}-'),  # -- and --- are TeX ligatures
+    (',', '{},'),  # ,, is a TeX ligature in T1 encoding
```

Comment:
maybe this is pushing too far doing that? , ,  in non  literal text is probably a typo anyhow. And this change forced to adapt a few tests.

Reply:
Because commas are very frequent in text, escaping to  `\sphinxcomma{}` is not very enticing. It will augment size of tex file. Overall I am actually tempted to do nothing with the comma, because avoiding the `,,` T1-ligature does not seem important. Visually the ligature looks like `,,` except it is a single character hence copying pasting from PDF will not give back `,,`. +1 for actually not doing anything with comma.

Reply:
Done, the comma is not escaped anymore.